packages/core/src/config/load.ts

```diff
@@ -50,6 +54,23 @@ export async function loadConfig(
       }] : []));
     }
   }
+
+  if (rawConfig.lint?.extends) {
+    rawConfig.lint = await resolveExtends({
+      lintConfig: rawConfig?.lint,
+      configPath,
+      resolve: rawConfig.resolve,
+    });
+  }
+
+  if (rawConfig.apis) {
+    rawConfig.apis = await resolveApis({
+      apis: rawConfig.apis,
+      configPath,
+      resolve: rawConfig.resolve,
+    });
+  }
```

Comment:
Please consider removing the `if` statements and use constants:
```suggestion
const lint = rawConfig.lint?.extends 
  ? await resolveExtends({
      lintConfig: rawConfig?.lint,
      configPath,
      resolve: rawConfig.resolve,
    })
    : rawConfig.lint;
  
  const apis = rawConfig.apis
    ? await resolveApis({
      apis: rawConfig.apis,
      configPath,
      resolve: rawConfig.resolve,
    })
    : rawConfig.apis
    
```
It would allow avoiding the mutations. And then you can simply use the properties below like this:
```
return new Config(
    {
      ...rawConfig,
      ...apis,
      lint: {
        ...lint,
        plugins: [...(rawConfig?.lint?.plugins || []), defaultPlugin], // inject default plugin
      },
    },
    configPath,
  );
}
```